CHANGELOG.md

```diff
@@ -48,6 +48,7 @@
 * [ENHANCEMENT] Ingester: Add two metrics tracking resource utilization calculated by utilization based limiter: #5496
   * `cortex_ingester_utilization_limiter_current_cpu_load`: The current exponential weighted moving average of the ingester's CPU load
   * `cortex_ingester_utilization_limiter_current_memory_usage_bytes`: The current ingester memory utilization
+* [ENHANCEMENT] Distributor: vendor update OpenTelemetry translator for exponential to Prometheus native histograms to implement downscale of exponential histograms that are over the native histogram scale limit of 8. #5532
```

Comment:
[nit] Should we say what the user-visible change is and leave out the implementation detail? Something like this:

```suggestion
* [ENHANCEMENT] Distributor: add support for ingesting exponential histograms that are over the native histogram scale limit of 8 in OpenTelemetry format by downscaling them. #5532
```

Reply:
yes, that's better, thank you!